arduinoTest/arduinoEcho/arduinoEcho.ino

```diff
@@ -2,7 +2,7 @@ const int ledPin = 11;
 
 void setup() {
   pinMode(ledPin, OUTPUT);
-  Serial.begin(9600);
+  Serial.begin(115200);
```

Comment:
@reconbot I tried to make sense of stress.js and requiresComPort.js, but since neither is commented, and I'm not familiar with the testing frameworks they used, I hit a dead end (and remembered, that's why I wrote this test in the first place). So I changed the baud rate back to 9600 so my test will work with those tests unmodified, I assume. I was able to test requiresComPort.js, which mocha validated at both 9600 and 115200bps, but couldn't do much with stress.js. Sorry.  

The PR should pass with the latest changes. And if anyone more knowledgeable wants to add some comments to stress.js and requiresComPort.js, I'll gladly test again.